workflow/applications/gfs_forecast_only.py

```diff
@@ -47,6 +47,9 @@ def _get_app_configs(self):
         if self.do_awips:
             configs += ['awips']
 
+        if self.do_npoess:
+            configs += ['npoess']
```

Comment:
Based on the ecflow trigger, I don't think this job can run in a forecast-only mode.

Reply:
Let's wait on Walter and/or Kate's feedback. I don't know anything about this task.

Reply:
Rahul is correct. There is no `postanl` job in forecast-only, so no `npoess`.